src/Components/Server/src/Circuits/RemoteJSRuntime.cs

```diff
@@ -59,22 +59,21 @@ protected override void EndInvokeDotNet(DotNetInvocationInfo invocationInfo, in
                     errorMessage += $". For more details turn on detailed exceptions in '{nameof(CircuitOptions)}.{nameof(CircuitOptions.DetailedErrors)}'";
                 }
 
-                EndInvokeDotNetCore(invocationInfo.CallId, success: false, errorMessage);
+                _clientProxy.SendAsync("JS.EndInvokeDotNet",
```

Comment:
We didn't do this before but do we need to validate `IsInitialized` here?

Reply:
TBH I'm not sure what useful thing we could do if `IsInitialized` was false. The code as-is would end up throwing a nullref, but I'm not sure how much it would help to throw a different error. As far as I know there's no case where this can happen, or at least it doesn't seem to have been a problem in any case so far.

Since the existing setup wasn't troubling anyone, I propose to keep this aspect of it as-is.

Reply:
That sounds sensible to me. It looks like the `IsInitialized` attribute was added recently as part of the error boundary work anyways which would explain why it wasn't used here.